### DISCUSSION TITLE
Spotted eDEX-UI on a company website 😂

### DISCUSSION THREAD

**Discussion by tianalemesle** [GitSquared/edex-ui#1029] [General]
[Arete Incident Response](https://www.areteir.com/) is a proxy company for corporations to quietly pay ransoms to hackers.

Did you too found eDEX-UI on other company websites ? Tell us !

**Comment by GitSquared** [comment]
I guess that's another milestone achieved :smile: next up, being featured in a movie?

Plus, it looks like they did a whole photoshoot with it... very interesting. Thanks for linking it here!

<details>
<summary>picture for future reference (in case the company deletes it)</summary>

![image](https://user-images.githubusercontent.com/24496417/107532871-9f917380-6bbe-11eb-9e2d-351c01ecef6b.png)
</details>

**Reply by tianalemesle** [reply]
> another milestone achieved

Definitely xD

> next up, being featured in a movie?

I'm looking forward to it :D

> Thanks for linking it here!

You're welcome !

**Reply by GitSquared** [reply]
Upon closer inspection, it looks like they used a screenshot from [this osxdaily article](http://osxdaily.com/2019/01/11/pretend-to-be-a-movie-hacker-with-edex-ui-terminal-emulator/). It comes up on the second row when searching for "hacker interface" images on big G. Interesting...

**Reply by tianalemesle** [reply]
Nice finding ! That makes this company even more guilty IMO :joy:

**Reply by wrac4242** [reply]
nice hacker style terminal, makes them seem experienced, similarly to the original tron inspiration. It looks cool, with thingies on, and no one questions thingies
